test/sms_nurse.test.js

```diff
@@ -177,29 +199,97 @@ describe("app", function() {
             it("should reverse their opt out status", function() {
                 return tester
                     .setup.user.addr("27820001003")
-                    .input("START")
+                    .inputs("START")
                     .check.interaction({
                         state: "states_opt_in",
                         reply:
                             "Thank you. You will now receive messages from us again. " +
                             "If you have any medical concerns please visit your nearest clinic"
                     })
+                    .check(function(api) {
+                        utils.check_fixtures_used(api, [52, 54, 182, 240]);
+                    })
                     .run();
             });
         });
 
         describe("when the user sends a different message", function() {
-            it("should tell them how to opt out", function() {
-                return tester
-                    .setup.user.addr("27820001003")
-                    .input("help")
-                    .check.interaction({
-                        state: "state_unrecognised",
-                        reply:
-                            "We do not recognise the message you sent us. Reply STOP " +
-                            "to unsubscribe or dial nurse_ussd_channel for more options."
-                    })
-                    .run();
+            
+            describe("when the message is received between 08:00 and 16:00", function() {
+                it("should log a support ticket", function() {
+                    return tester
+                        .setup.config.app({
+                            // friday during working hours
+                            testing_today: '2014-04-04 09:07:07'  // GMT+0200 (SAST)
+                        })
+                        .setup.user.addr('27820001003')
+                        .inputs('DONUTS')
+                        .check.interaction({
+                            state: 'states_default',
+                            reply:
+                                'Thank you for your message, it has been captured and you will ' +
+                                'receive a response soon. Kind regards. NurseConnect.'
+                        })
+                        .run();
```

Comment:
It's probably a good idea to add a `check_fixtures_used` line here and in the other tests to make sure we're using the expected fixture